pkg/cmd/create/create.go

```diff
@@ -436,6 +436,18 @@ func (o *Options) createPreviewNamespace() (string, error) {
 	return prNamespace, nil
 }
 
+func findAllServiceNamesInNamespace(client kubernetes.Interface, namespace string) ([]string, error) {
```

Comment:
Would be good to add a unit test.

Reply:
sure : )

Reply:
Should i make this function private?

Reply:
or should I shift this function in helpers?

Reply:
No need to make it public, you can test it indirectly in the Run function.

Reply:
okai

Reply:
The function is getting tested here already
https://github.com/jenkins-x-plugins/jx-preview/blob/main/pkg/cmd/create/create_test.go#L37